src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services6.yml

```diff
@@ -18,6 +18,8 @@ services:
         calls:
             - [ setBar, [ foo, '@foo', [true, false] ] ]
     alias_for_foo: '@foo'
+    another_third_alias_for_foo:
```

Comment:
This alias is before `another_alias_for_foo` because of an issue with the `YamlDumper`. See https://github.com/symfony/symfony/pull/21220